src/App/App.js

```diff
@@ -10,7 +10,7 @@ import store from '../Redux/store';
 import RegistrationComponent from './Components/Registration/Registration';
 import ForgotPassword from './Components/PasswordReset/ForgotPassword';
 import ResetPassword from './Components/PasswordReset/ResetPassword';
-import CreateArticle from '../App/Components/CreateArticle/CreateArticle';
+import CreateArticle from './Components/CreateArticle/CreateArticle';
```

Comment:
@nkpremices I need some context here. Why do you have a `CreateArticle` folder? What other components are/will be inside that folder?

What do you think about having an Articles folder that then holds all components related to articles e.g `CreateArticle` etc?

Reply:
The Idea of having only one folder holding all components related to articles..
`Create article` is already merged on develop @MemunaHaruna. Now when rebasing, I had an `eslint` issue on this file. The path that @truestbyheart used was not good. He was actually going out of a folder then enters in again. So I had to fix that on this line.